tests/test_oauth.yaml

```diff
@@ -0,0 +1,39 @@
+---
+- config:
+    - testset: 'Oauth API'
+
+- test:
+    - name: 'Get request returns a 405'
+    - url: '/oauth2/v1/access_token'
+    - method: 'GET'
+    - expected_status: [405]
```

Comment:
Yes, that stub implementation returns a 405 when the method is not defined in the Swagger document (there's no `GET` defined for that endpoint!)